test/e2e/scenarios/ACL/acl_create_update.yaml

```diff
@@ -74,12 +74,15 @@ steps:
       spec:
         userNames:
           - userone$RANDOM_SUFFIX
+        tags:
+          - key: "test_key_1"
+            value: "test_value_1"
```

Comment:
Curious here, how are you asserting that these tags got applied to the ACL?

Reply:
I had the same comment in another PR - we aren't validating that the AWS resource actually has the tags.

Reply:
Resolved in PR #65